sdl/v1_test.go

```diff
@@ -16,14 +16,30 @@ const (
 	randStorage uint64 = 1 * unit.Gi
 )
 
-func Test_v1_Parse_docs(t *testing.T) {
-	sdl, err := sdl.ReadFile("../x/deployment/testdata/deployment.yaml")
+func Test_v1_Parse_Deployments(t *testing.T) {
+	SDLv1, err := sdl.ReadFile("../x/deployment/testdata/deployment.yaml")
```

Comment:
lowercase if you must.  maybe better to use aliased import instead - `import sdlv1 "github.com/ovrclk/akash/sdl/v1"`

Reply:
I agree, it makes sense to split apart `sdl` package into `sdl/v1` `sdl/vN` etc. It's imported in a lot of places though, so I'd rather not refactor all of that for this PR. I've updated the import alias and reset the variable used to `sdl`.

Shall I open up an issue for splitting up versioned parts of `sdl` into `sdl/v...` packages?